src/jvm/test/resources/expected_logs/internal_bug_report.txt

```diff
@@ -6,6 +6,6 @@ Exception stacktrace:
 java.lang.IllegalStateException: Internal bug
 	at org.jetbrains.kotlinx.lincheck.util.InternalLincheckExceptionEmulator.throwException(InternalLincheckExceptionEmulator.kt:23)
 	at org.jetbrains.kotlinx.lincheck_test.representation.InternalLincheckBugTest.operation2(InternalLincheckBugTest.kt:38)
-	at org.jetbrains.kotlinx.lincheck.runner.TestThreadExecution301.run(Unknown Source)
-	at org.jetbrains.kotlinx.lincheck.runner.FixedActiveThreadsExecutor.testThreadRunnable$lambda$8(FixedActiveThreadsExecutor.kt:149)
+	at org.jetbrains.kotlinx.lincheck.runner.TestThreadExecution9924.run(Unknown Source)
```

Comment:
It's fine not to change the output. Please rebase onto `develop` and check whether the test outputs should be changed. I would expect that all (or almost all) test outputs remain the same.

Reply:
The only thing is changed here is TestThreadExecution number, generated by jvm, so it's not a change at all.

Reply:
Fixed